features/events/syslog/pom.xml

```diff
@@ -260,5 +266,30 @@
         </exclusion>
       </exclusions>
     </dependency>
+    
+    <!-- HZN-742 -->
+    <dependency>
+	 <groupId>org.apache.curator</groupId>
+	 <artifactId>curator-test</artifactId>
+	 <version>2.0.0-incubating</version>
+	 <scope>test</scope>
```

Comment:
You need to exclude log4j:log4j from this curator dependency. Since it is the only thing new here, it must be causing the log4j dependency problems. Just add an <exclusions><exclusion>...</exclusion></exclusions> block with the log4j groupId and artifactId.